data/json/items/ammo.json

```diff
@@ -141,7 +141,7 @@
     "category": "spare_parts",
     "price": 648,
     "price_postapoc": 10,
-    "name": { "str": "aluminum foil" },
+    "name": { "str_sp": "aluminum foil" },
```

Comment:
```suggestion
    "name": { "str_sp": "pieces of aluminum foil" },
```

Reply:
```suggestion
    "name": { "str": "piece of aluminum foil", "str_pl": "pieces of aluminum foil" },
```

Reply:
Did you check in your game to see if the game accepts that? What do the maintainers think of this style?

Reply:
Is your concern re size on the screen, or what? Existing ones have things like `{ "str": "chunk of sulfur", "str_pl": "chunks of sulfur" }`. I'll check it when I can, but I'm going to be occupied with teaching this coming week.

Reply:
Oh, if existing items in the game have the "name" members all on the same line, and the game works, that's fine. My concern was whether the game would know what to do with it.

Reply:
Oh. Yes, that's how one is supposed to do it (it may be put on the next line in terms of a newline (as opposed to another "name":), but that the JSON formatter can take care of).

Reply:
I'd like to keep the original "str_sp", to reduce translator burden.

Reply:
Yeah. I'm OK with that, from below.